R/OWS-methods-parsing.R

```diff
@@ -289,14 +290,14 @@ parseOwsRange <- function(obj) {
   .filtered <- c()
   .names <- c()
   for (.x in .temp) {
-    if(xmlName(.x) == childrenName && includeNamed) {
-      .filtered <- c(.filtered, .x)
+    if(includeNamed && xmlName(.x) == xmlTagName) {
       if(verbose) cat("[.filterXmlChildren] Added", xmlName(.x), "\n")
+      .filtered <- c(.filtered, .x)
       .names <- c(.names, xmlName(.x))
     }
-    else if(!includeNamed && xmlName(.x) != childrenName) {
-      .filtered <- c(.filtered, .x)
+    else if(!includeNamed && xmlName(.x) != xmlTagName) {
       if(verbose) cat("[.filterXmlChildren] Added", xmlName(.x), "\n")
+      .filtered <- c(.filtered, .x)
```

Comment:
> Why change the order here and log before the assignment? 

Because checking the boolean before calling the function might be faster. No other thoughts.

Updated the section a bit, so please take another look, when I pushed my changes to the PR.